Presentation.Web/Web.config

```diff
@@ -4,363 +4,365 @@
+			</setting>
+			<setting name="SsoServiceProviderId" serializeAs="String">
+				<value>https://kitos-local.strongminds.dk</value>
+			</setting>
+		</Presentation.Web.Properties.Settings>
+	</applicationSettings>
+	<!--<system.diagnostics>
     <trace autoflush="true">
       <listeners>
         <add name="SerilogTrace" type="Presentation.Web.Infrastructure.Tracelistener.SerilogTraceListener, Presentation.Web" />
         <remove name="Default" />
       </listeners>
     </trace>
   </system.diagnostics>-->
-  <Federation sessionType="dk.nita.saml20.Session.InProcSessionStoreProvider, dk.nita.saml20" xmlns="urn:dk.nita.saml20.configuration">
-    <SessionTimeout>30</SessionTimeout>
-    <PreventOpenRedirectAttack>true</PreventOpenRedirectAttack>
-    <SigningCertificate findValue="1793d097f45b0acea258f7fe18d5a4155799da26" storeLocation="LocalMachine" storeName="My" x509FindType="FindByThumbprint" />
-    <MetaDataShaHashingAlgorithm>SHA256</MetaDataShaHashingAlgorithm>
-    <AllowedAudienceUris>
-      <Audience>https://kitos-local.strongminds.dk</Audience>
-    </AllowedAudienceUris>
-    <Actions>
-      <clear />
-      <add name="SetSamlPrincipal" type="dk.nita.saml20.Actions.SamlPrincipalAction, dk.nita.saml20 " />
-      <add name="Redirect" type="dk.nita.saml20.Actions.RedirectAction, dk.nita.saml20" />
-    </Actions>
-  </Federation>
-  <SAML20Federation xmlns="urn:dk.nita.saml20.configuration" xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata">
-    <MinimumAssuranceLevel>2</MinimumAssuranceLevel>
-    <ShowError>true</ShowError>
-    <ServiceProvider id="https://kitos-local.strongminds.dk" server="https://localhost:44300">
-      <ServiceEndpoint localpath="Login.ashx" type="signon" redirectUrl="SSO" index="0" />
-      <ServiceEndpoint localpath="Logout.ashx" type="logout" redirectUrl="SSO" index="1" />
-      <ServiceEndpoint localpath="Logout.ashx" type="soaplogout" errorBehaviour="throwexception" />
-      <ServiceEndpoint localpath="Metadata.ashx" type="metadata" />
-      <md:ContactPerson contactType="administrative">
-        <md:Company>STRONGMINDS</md:Company>
-        <md:EmailAddress>admin@strongminds.dk</md:EmailAddress>
-      </md:ContactPerson>
-    </ServiceProvider>
-    <RequestedAttributes>
-      <att name="urn:oid:0.9.2342.19200300.100.1.1" isRequired="true" />
-      <!--sAMAcountName, AD username-->
-    </RequestedAttributes>
-    <NameIdFormat>urn:oasis:names:tc:SAML:1.1:nameid-format:X509SubjectName</NameIdFormat>
-    <IDPEndPoints metadata="idp-metadata">
-      <add id="https://saml.adgangsstyring.eksterntest-stoettesystemerne.dk" ShaHashingAlgorithm="SHA256" default="true" />
-    </IDPEndPoints>
-    <CommonDomain enabled="false" localReaderEndpoint="https://pfs04/demo/cdcreader.ashx" />
-  </SAML20Federation>
-  <!--
+	<Federation sessionType="dk.nita.saml20.Session.InProcSessionStoreProvider, dk.nita.saml20" xmlns="urn:dk.nita.saml20.configuration">
+		<SessionTimeout>30</SessionTimeout>
```

Comment:
@a-naskret You need to change your "tab" settings to use whitespaces, not tabs :-)